gcp/modules/cert-manager/resources/issuers.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: certmanager.k8s.io/v1alpha1
-kind: Issuer
+kind: ClusterIssuer
```

Comment:
I'm a little confused about this change, and where we want an Issuer vs a ClusterIssuer. This comment is just a heads up that I will ask you to explain it to me a bit :).

Reply:
ClusterIssuer has broader permission set to create resources in any namespace. This supports workflow with single cert-manager deployment in the cluster (and, let's say to be able to get Letsencrypt certs for gpii and monitoring at the same time).